packages/@react-aria/calendar/src/useCalendarBase.ts

```diff
@@ -146,6 +146,8 @@ export function useCalendarBase(props: CalendarPropsBase & DOMProps, state: Cale
       'aria-disabled': isDisabled || null,
       'aria-labelledby': labelProps['aria-labelledby'],
       'aria-describedby': selectedDateDescription ? captionId : null,
+      'aria-colcount': 7,
+      'aria-rowcount': state.weeksInMonth + 1,
```

Comment:
Are these necessary? I thought if not provided they'd be computed automatically based on the DOM structure.